connection.go

```diff
@@ -232,6 +233,17 @@ func (b *ConnectionBuilder) URL(url string) *ConnectionBuilder {
 	return b.AlternativeURL("", url)
 }
 
+// Region builds the base URL of an API gateway based on an OCM region name. The default is
+// `https://api.%s.openshift.com`. Where `%s` is replaced by the region name.
+//
+// This method should be used in-lieu of URL when connecting to a regionalized API gateway.
+func (b *ConnectionBuilder) Region(region string) *ConnectionBuilder {
+	if b.err != nil {
+		return b
+	}
+	return b.AlternativeURL("", fmt.Sprintf(FormattedDefaultRegionURL, region))
```

Comment:
this would return `https://api..opeshift.com` for commercial prod (where empty region id is meaningful)

Reply:
The SDKs default URL is prod. Not attaching region or URL defaults to prod. No need to support commercial prod with the Region attribute.

Reply:
https://github.com/openshift-online/ocm-sdk-go/blob/main/connection.go#L133

Reply:
updated to short-circuit when a value is not provided